### ISSUE TITLE
Other npm packages have references pages.css

### ISSUE THREAD

**Issue by LQ0611** [jonathantneal/sanitize.css#225]
I encountered a problem：when i used other other npm packages ,and it have references sanitize.css/pages.css ,when i  'yarn' , and install 13.0.0, but  cannot find module ‘sanitize.css/page.css’

**Comment by LQ0611** [comment]
When updating, is it compatible with previous versions

**Comment by yoyo837** [comment]
MAJOR is not compatible, very normal.

**Comment by stheine** [comment]
An issue is here in postcss-normalize, where they did not bind the include to a specific or even major version:
https://github.com/csstools/postcss-normalize/issues/58

**Comment by KutnerUri** [comment]
yup this just broke my package because of the indirect dependency through `postcss-normalize`. :(

**Comment by madhu5428** [comment]
> I encountered a problem：when i used other other npm packages ,and it have references sanitize.css/pages.css ,when i 'yarn' , and install 13.0.0, but cannot find module ‘sanitize.css/page.css’

Do you know how to resolve it by using npm?
